bokehjs/src/coffee/core/ui_events.coffee

```diff
@@ -130,13 +130,13 @@ export class UIEvents
         if view?
           if view.model.cursor?
             cursor = view.model.cursor()
-          if active_inspectors?
```

Comment:
@mattpap do you know why the tests didn't catch the bug though? It seems the test interpreter eval-ed that line differently than a browser